server/api/playerAPI/player.js

```diff
@@ -67,6 +67,19 @@ router.get('/allPlayers/:roomId', async (req, res, next) => {
     next(error);
   }
 });
+/// ********************** WRITTEN BY JOSH ********************** ///
+//GET --- gets all players
+//✔ works
+router.get('/allPlayers', async (req, res, next) => {
```

Comment:
Can we rename this get to be `/` and not `allPlayers` ? 
Having a route like:
`router.get('/allPlayers' ...`
and
`router.get('/:playerId' ...`
is dangerous.

I realize with our setup using firebase uids this case shouldn't happen but - if the player had the playerId of allPlayers (especially because we are using strings for ids), this would get confused and return all the players, and we could never get the specific player's info.